index.js

```diff
@@ -18,4 +18,242 @@ window.onclick = function exit(event) {
+    tecnologies: {
+      tech1: 'Codekit',
+      tech2: 'Github',
+      tech3: 'JavaScript',
+      tech4: 'Terminal',
+      tech5: 'Bootstrap',
+    },
+    linklive: 'https://brhanuhailu.github.io/',
+    linksource: 'https://github.com/brhanuhailu/portifolio',
+  },
+  {
+    id: 2,
+    url: './images/project2.png',
+    title: 'Keeping track of hundreds of components',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    tecnologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+      tech4: 'html',
+    },
+    linklive: 'https://brhanuhailu.github.io/',
+    linksource: 'https://github.com/brhanuhailu/portifolio',
+  },
+  {
+    id: 3,
+    url: './images/project4.png',
+    title: 'Build a CryptoCurrency Tracker App',
+    description: 'The idea is great and you can actually put this app into App Store or Google PlayStore as React Native allows you to create cross-platform apps. You can use React.js, React Native, and Firebase to build the course, where Firebase can be used ',
+    tecnologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+      tech4: 'html',
+    },
+    linklive: 'https://brhanuhailu.github.io/',
+    linksource: 'https://github.com/brhanuhailu/portifolio',
+  },
+  {
+    id: 4,
+    url: './images/project1.png',
+    title: 'Keeping track of hundreds of components',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum is simply dummy text of the printing and typesetting industry',
+    tecnologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+      tech4: 'html',
+    },
+    linklive: 'https://brhanuhailu.github.io/',
+    linksource: 'https://github.com/brhanuhailu/portifolio',
+  },
+  {
+    id: 5,
+    url: './images/project2.png',
+    title: 'Keeping track of hundreds of components',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ths helps you to avoid plagiarisum and support to respect the intellectual property right',
+    tecnologies: {
+      tech1: 'Ruby on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+      tech4: 'html',
+    },
+    linklive: 'https://brhanuhailu.github.io/',
+    linksource: 'https://github.com/brhanuhailu/portifolio',
+  },
+];
+const workdetailsSection = (content) => `
+    <span class="close-details">&times;</span>
+    <img class="imag-details" src=${WorkDatas[content].url} alt="Image details">
+    <h2 class="project-name">${WorkDatas[content].title}</h2>
+    <ul class="detail-tags">
+        <li><a href="">${WorkDatas[content].tecnologies.tech1}</a></li>
+        <li><a href="">${WorkDatas[content].tecnologies.tech2}</a></li>
+        <li><a href="">${WorkDatas[content].tecnologies.tech3}</a></li>
+        <li><a href="">${WorkDatas[content].tecnologies.tech4}</a></li>
+    </ul>
```

Comment:
- [ ] I have noticed that the languages are not centered. If you are keen you can see that the languages on the popup are vertically centered. Kindly ensure that they meet the Figma design template ☺️ .

Your design 😎 
![lang pop](https://user-images.githubusercontent.com/99394374/216003235-23c746c6-0cc6-4b77-bf56-004a5fcd2c0e.PNG)

 Figma design 😎 
![lang pop fig](https://user-images.githubusercontent.com/99394374/216003332-22785002-0bd9-4ee8-bc08-279c3d781df2.PNG)

Reply:
Dear @AlphaNtihinduka 
Thank you for your constructive comments, but We have one question to be cleared on the  " You need to use a JavaScript array to store all of the information for all projects" all the project are in work sections. there are no project out of the work section.  Personally , I am little bit confused about that comment. may i get some information please?

Reply:
Fixed.